examples/guestbook-go/README.md

```diff
@@ -169,15 +169,12 @@ redis-slave    name=redis,role=slave    name=redis,role=slave    10.0.234.24
 To play with the service itself, find the external IP of the load balancer:
 
 ```shell
-$ cluster/kubectl.sh get services guestbook -o template --template='{{index . "publicIPs"}}'
-current-context: "kubernetes-satnam_kubernetes"
-Running: cluster/../cluster/gce/../../_output/dockerized/bin/linux/amd64/kubectl get services guestbook -o template --template='{{.spec.publicIPs}}'
-[104.154.63.66]$
-
+$ cluster/kubectl.sh get services guestbook -o template --template='{{(index .status.loadBalancer.ingress 0).ip}}'
+23.251.156.224$
 ```
 and then visit port 3000 of that IP address e.g. `http://104.154.63.66:3000`.
```

Comment:
This line references the IP in the old example - either make your IP be 104... or make this be 23....